index.html

```diff
@@ -2231,8 +2228,9 @@ <h3>Extensibility</h3>
           </li>
           <li>
 Support multiple types of cryptographic proof formats through the use of
-Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of signature suites
-listed in the Linked Data Cryptographic Suites Registry [[?LDP-REGISTRY]]
+Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
+suites listed in the Verifiable Credential Specifications Directory
+[[?VC-SPECS]].
```

Comment:
This is kinda using the vc-specs-dir as a registry for:

https://w3id.org/security#DataIntegrityProof

cryptosuites property.

Reply:
@brentzundel @Sakurann @msporny I worry a bit about this,  if we will encounter complaints in the future.

Feels like we should either omit [[VC-JWT]] and [[VC-DATA-INTEGRITY]] and just point to the registry, or we should omit the registry pointer and just point to them.

We are kinda creating a 2 class system here.

Reply:
We could probably file an issue to track this and move on with the PR.